editoast/editoast_derive/src/modelv2.rs

```diff
@@ -187,8 +220,48 @@ impl From<ModelFieldOption> for ModelField {
             identifier: value.identifier,
             preferred: value.preferred,
             primary: value.primary,
-            json: value.json,
-            geo: value.geo,
+            transform: FieldTransformation::from_args(
+                value.remote,
+                value.json,
+                value.geo,
+                value.to_string,
+            )?,
```

Comment:
To get a better error spaning.

```suggestion
            )
            .map_err(|e| e.with_span(&builder_ident))?,
```